lib/openQAcoretest.pm

```diff
@@ -34,8 +34,9 @@ sub upload_openqa_logs {
     my @logs = split m/\n/, script_output q{find /var/lib/openqa -name autoinst-log.txt};
     @logs and get_log "(cat @logs ||:)" => 'autoinst-log.txt';
     get_log '(find /var/lib/openqa/pool/ /var/lib/openqa/testresults/ ||:)' => 'find.txt';
-    script_run 'tar -cvzhf testresults.tar.gz /var/lib/openqa/testresults/';
-    upload_logs 'testresults.tar.gz';
+    # do not assert for tar as it might return 1 due to the -h flag when dereferencing broken symlinks
+    script_run q{tar czhvf /tmp/testresults.tar.gz -C /var/lib/openqa/ testresults};
+    upload_logs "/tmp/testresults.tar.gz" if (script_run("test -f /tmp/testresults.tar.gz") == 0);
```

Comment:
```suggestion
    script_run q{tar cahvf /tmp/testresults.tar.xz -C /var/lib/openqa/ testresults};
    upload_logs "/tmp/testresults.tar.xz" if (script_run("test -f /tmp/testresults.tar.gx") == 0);
```

Reply:
So we do prefer `xz` over `gzip`? I guess it should be just on this place so I can definitely easily change that.